docs/features/battery.mdx

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 6
+---
+import Screenshot from "../../src/components/Screenshot";
+
+# Hausbatterie
+
+Viele Photovoltaiksysteme werden heute mit einer Hausbatterie kombiniert.
+Die Batterie speichert überschüssige Solarenergie und stellt sie bei Bedarf wieder zur Verfügung.
+
+evcc bringt einige Funktionen mit, um das Zusammenspiel zwischen dem Elektroauto und der Hausbatterie zu optimieren.
+Ist der Batterie- bzw. Hybridwechselrichter konfiguriert, kennt evcc den Ladestand der Hausbatterie und kann diesen bei der Laderegelung des Elektroautos berücksichtigen.
+
+## Wohin geht die Sonnenenergie zuerst?
+
+Hausbatterien reagieren sehr schnell auf verfügbaren Überschuss und nehmen diese Energie auf.
+Im Standardfall würde daher erst die Hausbatterie komplett geladen werden, bevor freier Überschuss in das Elektroauto fließt.
+
+Besitzt du einen größeren Speicher, kann es aber auch sinnvoll sein, bereits früher mit der Ladung des Elektroautos zu beginnen.
+Dieses Verhalten kannst du im **Batterieeinstellungen**-Dialog konfigurieren.
+
+<Screenshot name="features/screenshots/battery-navigation" caption="Screenshot der Hauptnavigation." />
+
+Hier stellst du ein, bis zu welchem Ladestand die Hausbatterie prioritär geladen werden soll und ab wann das Elektroauto den Überschuss nutzen darf.
+Mit der unteren Grenze legst du fest, wie viel Energie für Hausverbrauch reserviert bleibt (1) und in welchem Bereich das Fahrzeug Vorrang hat (2).
+
+<Screenshot name="features/screenshots/battery-priority" caption="Priorität Haus und Auto" />
+
+Beispiel: Du setzt die Grenze auf 50%. Das bedeutet, dass Sonnenenergie erstmal in die Hausbatterie fließt.
+Erreicht die Batterie einen Ladestand von 50%, werden Ladepunkte, die sich im PV-Modus befinden, freigegeben.
+Die Ladeleistung wird so reguliert, dass die Hausbatterie ihren Ladestand hält.
+Ist kein Fahrzeug angeschlossen, geht überschüssige Sonnenenergie natürlich weiterhin in die Hausbatterie.
+
+## Batterieunterstütztes Laden
+
+Im PV-Modus versucht evcc die Ladung so zu steuern, dass nur Sonnenenergie direkt genutzt wird.
+Ein Entladen der Hausbatterie wird dabei vermieden.
+Mit der Funktion **Batterieunterstütztes Laden** kannst du einen Teil deines Hausspeichers aber explizit für das Laden des Elektroautos freigeben.
+
+Die obere Grenze legt fest, welcher Teil (3) der gespeicherten Energie ins Elektroauto rüberfließen darf.
+
+<Screenshot name="features/screenshots/battery-buffer" caption="Batterieunterstütztes Laden" />
+
+Beispiel: Du gibst den Bereich oberhalb von 75% für Batterieunterstütztes Laden frei.
+Dein Hausspeicher hat sich über den Tag bereits auf 100% gefüllt.
+Du steckst dein Elektroauto im PV-Modus an.
+Zusätzlich zum verfügbaren Überschuss werden jetzt auch die freigegebenen oberen 25% der Hausbatterie genutzt, um das Elektroauto zu laden.
+Ist kein Überschuss mehr verfügbar und der Ladestand des Hausspeichers ist auf 75% gefallen, wird die Ladung gestoppt.
```

Comment:
Ist kein Überschuss mehr verfügbar, wird mit minimaler Leistung weitergeladen, bis der Ladestand des Hausspeichers auf 75% gefallen ist. In Abhängigkeit der Entladeleistung des Hausspeichers kann es hierbei auch zu Netzbezug kommen.